shared/constants/notifications.ts

```diff
@@ -1,2 +1,2 @@
 export const NOTIFICATION_HEIGHT = 620;
-export const NOTIFICATION_WIDTH = 360;
+export const NOTIFICATION_WIDTH = 400;
```

Comment:
I'm not entirely sure that this is the correct value that is used by the notifications window. On `main`, I'm seeing the notifications popup already open at `400px`. Maybe @MetaMask/confirmations or @FrederikBolding can clarify this?